core/lib/list_item_hunter.js

```diff
@@ -13,6 +13,7 @@
 var list_item_hunter = function () {
 
   var extend = require('util')._extend,
+    JSON = require('json5'),
```

Comment:
@james-nash Had to debate over this myself, and I'm still torn. @bmuenzenmeyer What are your thoughts?